.github/ISSUE_TEMPLATE/typescript-eslint-parser.md

```diff
@@ -0,0 +1,13 @@
+**What version of TypeScript are you using?**
```

Comment:
Do we want to include asking what version of ESLint they're using, if they're using it? Seems like the exception to this would be if someone using `typescript-estree` (like Prettier is).

Reply:
there is also issue template for `typescript-estree.md`

Reply:
Might be cool to leverage the multiple issue template feature with different templates for different packages (example [here](https://github.com/eslint/eslint/issues/new/choose)).